src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/models/CCJContent.java

```diff
@@ -82,4 +94,58 @@ public String getSignerName() {
     public String getSignerRole() {
         return signerRole;
     }
+
+    private void setAmountToPayByDefendant(
+        final InterestCalculationService interestCalculationService,
+        final Claim claim
+    ) {
+        CountyCourtJudgment countyCourtJudgment = claim.getCountyCourtJudgment();
+        if (countyCourtJudgment.getPaymentOption() == PaymentOption.INSTALMENTS) {
+            countyCourtJudgment.getRepaymentPlan().ifPresent(
+                repaymentPlan -> this.amountToPayByDefendant = formatMoney(repaymentPlan.getRemainingAmount())
+            );
+        } else if (countyCourtJudgment.getPaymentOption() == PaymentOption.FULL_BY_SPECIFIED_DATE) {
+            this.amountToPayByDefendant = calculateClaimAmountWithInterest(
+                interestCalculationService,
+                claim,
+                countyCourtJudgment.getPayBySetDate().get()
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS3655)